### PR TITLE
Create dedicated sponsors page.

### PR DESCRIPTION
**Pull request by dukebody**
Logos for all sponsors who hosted a meetup/pyladies/practice session on 2019 and others.

image

See https://docs.google.com/spreadsheets/d/1YARj7MPp_3wMMmmjCH72y10wsGGLD-1p8TB9VIv3JJg/edit#gid=0.

Is the order OK? I did semi-randomly.

### CODE REVIEW

**Comment by dukebody** [comment]
Adding the typeform one, hold on...

**Comment by jsoucheiron** [comment]
Could you add the new skyscanner logo?
https://brandfolder.com/skyscanner

El dg., 15 de des. 2019, 14:49, Israel Saeta Pérez <notifications@github.com>
va escriure:

**Comment by dukebody** [comment]
> Could you add the new skyscanner logo? https://brandfolder.com/skyscanner 

Done

**Comment by jsoucheiron** [comment]
Regarding the order I dont think there will be issues. If you wanna be sure
though, go alphabetical, this way you will avoid any possible argument

El dg., 15 de des. 2019, 15:44, Israel Saeta Pérez <notifications@github.com>
va escriure:

**Comment by ifosch** [comment]
Why is Codely there? There should also be JetBrains, IIRC. I also miss Kernel Analytics, Punt Multimedia... Not sure if there is anyone else missing.

**Comment by dukebody** [comment]
> Why is Codely there? There should also be JetBrains, IIRC. I also miss Kernel Analytics, Punt Multimedia... Not sure if there is anyone else missing.

Codely is there because it is listed as host for one of the PyLadies session in https://docs.google.com/spreadsheets/d/1YARj7MPp_3wMMmmjCH72y10wsGGLD-1p8TB9VIv3JJg/edit#gid=0.

Punt Multimèdia is there (FAB logo at the bottom).

Regarding Kernel Analytics, I believe they didn't host any PyBCN activity in 2019 and it is not an "active sponsor" right now, but I'm open to list them there as a thank you for all their awesome support in the past. :)

Regarding JetBrains, did they ever host any meetup or activity? When was the last time they sponsored with PyCharm licenses? IDK, listing all historical sponsors might be too much perhaps.

Which criteria should we use to decide who to include?

**Comment by rberenguel** [comment]
I think activity in the last 12 months is a very good one?

**Comment by DZPM** [comment]
Some questions and proposals:
- 1) should add info on what is a sponsor, otherwise it's not transparent, and looks suspicious ("are these companies paying these guys"?). How about a line, like: "We want to thank this organizations for providing venue or sponsorship in our events:"
- 2) how about companies who sponsored individual events, like pydata or pyday? If we do the "12 months" rule, we should add the ones for the PyDay 2019
- 3) why the "mailto:" tag, with our contact email in plain text? Not cool, it will get us lots of spam :cry: 
- 4) if the deadline is 12 months, should we remove the sponsor logo 12 months after the meetup they hosted? (too much work... let's do it just once a year, maybe?)

Thanks!

**Comment by rberenguel** [comment]
You are right. What about:

These are companies that have sponsored us recently (current or past year) by providing catering or hosting for any of our events, or in some other way that has helped us with our goal to widen the Python community around Barcelona

As for duration, the easiest is what I mention: current year +past year, which makes it easier to prune (once a year for the next year). I.e. on January we should have the sponsors for the previous year (2019) only, and then add any new ones.

**Comment by ifosch** [comment]
I would rephrase @rberenguel's proposal like the following:

> These are companies that have sponsored us recently, i.e. in the last two years, in some way that has helped us with our goal to widen the Python community around Barcelona.

I think it's ok to update the web page yearly, every January, f.e.

About the mailto, I agree this will get us too much spam. I think we could put a link to our twitter account and have them approach us that way.

About the sponsors for individual events, we already have their logos in the event page, which I think is what we promise in the sponsors prospection document, and, to me, is enough. Another option here would be to have different sections in this document, maybe also detailing what the sponsors provided in each case, but might end up being too redundant.

**Comment by DZPM** [comment]
- your description looks good :+1: 
- most people don't have twitter accounts. How about putting the email address in an embedded image?
- having different sections could make sense in the long term: one for universities, one for venues, one for sponsors...
- if we don't add sponsors from events, maybe let's add a link to that event? as in "Old sponsors for <PyDataBCN 2017>, <PyDayBCN 2018>, <PyDayBCN 2019>, ...)

**Comment by rberenguel** [comment]
Sí a todo!

**Comment by dukebody** [comment]
Implemented all requests.

For the mail link I used the address "pybcnREMOVETHIS@googlegroups.com". However I don't think putting there the real mail address would give us any significative spam, since Google Groups have anti-spam filters just like Gmail. For example, at https://pybcn.org/association/become-a-member/ we have a mail address for a group in clear text and I'm not aware of any spam on that group. But anyhow.
